.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: scala
+script:
+  - ./bin/runci.sh $CI_TEST
+jdk:
+ - oraclejdk8
+ env:
+   global:
+   - TRAVIS_NODE_VERSION="8"
+   matrix:
+     - CI_TEST: check
+     - CI_TEST: ci
+       CI_SCALA_VERSION: 2.11.12
```

Comment:
The 2.11 ones will need to be removed from the build matrix unless we add crossbuilding between scala version

Reply:
I propose we only build for 2.12.4 to begin with, we can add 2.11 cross-building if there is demand for it, but I don't right now a use-case where 2.12 is not OK.